### DISCUSSION TITLE
Escorted civilians ignore prohibited doors

### DISCUSSION THREAD

**Discussion by Xayrlen** [FakeFishGames/Barotrauma#12962] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Using shift + LMB, I prohibited reactor room from entering it due to volatile fulgurium rods present there. However, non-crew personnel doesn't abide by that order and get itself fried while walking around the sub. Right now you have to weld doors to prevent civilians from going inside.
![1](https://github.com/Regalis11/Barotrauma/assets/86978521/e02d7840-18d2-4f53-b170-032b09e51e23)
![2](https://github.com/Regalis11/Barotrauma/assets/86978521/f3a34a64-a4a7-42b5-8eea-df73c8e9d0de)
![3](https://github.com/Regalis11/Barotrauma/assets/86978521/ed73c8e8-484d-4de6-8b58-f24bdabd5a22)

### Reproduction steps

1. Start a single/multiplayer campaign;
2. Order to ignore any doors;
3. Take Civilian Escort mission, start traveling;
4. Watch civilians walk through ignored doors.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.1.19.3 (Treacherous Tides Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I wouldn't necessarily consider this a bug - escorted characters don't follow any orders you give to the crew. While we could make an exception in this case, I think it might lead to unintended side-effects. I think the escortees should be able to cause trouble (by wandering into places they shouldn't, taking weapons and such if they're mentally unstable or hostile), and it might be a little hard to draw the line on which ignore orders they should follow and which they shouldn't. One could also argue that keeping the escortees out of rooms they shouldn't go inside is part of the challenge in the escort missions. I think we might look into this, but I'll move this into discussions because imo it's more of a feature request to change the behavior of the escortees, rather than a bug report.
